programmers/난이도별/level01.두_정수_사이의_합/HyeonJeong.c

```diff
@@ -2,16 +2,23 @@
 #include <stdbool.h>
 #include <stdlib.h>
 
+int main(void){
+    printf("%d %d", &solution(a,b), 12L)
+}
+
 long long solution(int a, int b) {
     long long answer = 0;
 
-    if(a>b)
-        for(int i = 0; i <= a - b; i++)
-            answer += b + i;
-    else if(a<b)
-        for(int i = 0; i <= b - a; i++)
-            answer += a + i;
+    if(a != b){
+        int max = a > b ? a : b;
+        int min = a < b ? b : a;
+
+        for (int i = 0; i <= max - min; i++)
+        answer += min + i;
+    }
     else
+        // a와 b가 같은 경우는 둘 중 아무 수나 리턴
         answer = a;
+
     return answer;
```

Comment:
```suggestion
    if (a == b) return a;
    int max = a > b ? a : b;
    int min = a < b ? b : a;

    for (int i = 0; i <= max - min; i++)
        answer += min + i;

    return answer;
```

`if (a == b)`에 대해 검사 후 `return`을 하면 뒤에서 `else`를 사용하지 않아도 됩니다!

항상 코드를 어떻게 하면 더 쉽게, 간결하게 보여질 수 있을까에 대해 고민해보면 좋아요!

특히, 코드에 if, else가 많을 경우 이해하기가 어려워지는 경우가 많다보니 가능 하면 if, else가 코드에 나타나지 않도록 고민해보는 연습을 하면 좋습니다~ 아예 없을 순 없지만 최소화 시키는거죠!